src/player.ts

```diff
@@ -249,6 +269,8 @@ export default class Player extends EventEmitter {
     for (const event of this.events[Operator.Income]) {
       this.gainRewards(event.rewards);
     }
+
+    //TODO : reactivate events 
```

Comment:
This is simple. The events that can be reactivated are only those of `Operator.Activate`

```
for (const event of this.events[Operator.Activate]) {
  event.activated = false;
}
```

Reply:
done